yarn-project/node-keystore/src/loader.ts

```diff
@@ -284,38 +288,43 @@ export function mergeKeystores(keystores: KeyStore[]): KeyStore {
  * @param attester The attester configuration in any supported shape.
  * @returns Array of string keys used to detect duplicates.
  */
-function extractAttesterKeys(attester: unknown): string[] {
+function extractAttesterAddresses(attester: unknown): string[] {
```

Comment:
This function used to stringify the attester for deduplication reasons. Private keys/addresses/mnemonic/JSON encrypted files + passwords, all were fair game for being added to a `Set<string>` and logged to stdout if a duplicates were found.

The new function instead derives the address and uses that for deduplication. If something can not be readily transformed to an Eth address it is ignored and duplication is checked at run time.